mqtt_client/src/MqttClient.ros2.cpp

```diff
@@ -200,10 +309,16 @@ void MqttClient::loadParameters() {
     declare_parameter(fmt::format("bridge.ros2mqtt.{}.mqtt_topic", ros_topic), rclcpp::ParameterType::PARAMETER_STRING, param_desc);
     param_desc.description = "whether to publish as primitive message";
     declare_parameter(fmt::format("bridge.ros2mqtt.{}.primitive", ros_topic), rclcpp::ParameterType::PARAMETER_BOOL, param_desc);
+    param_desc.description = "Explicitly set the ros message type";

```

Comment:
Please make descriptions here and parameter descriptions in the README consistent.